R/colby_constructors.R

```diff
@@ -1210,27 +1147,21 @@ get_acolvar_vars <- function(lyt) {
   }
 }
 
-
-#' Generate Rows Analyzing Different Variables Across Columns
-#'
-#' @inheritParams  lyt_args
-#'
-#' @param afun function or list. Function(s) to be used to calculate the values
-#'   in each column.  The list will be repped out as needed and matched by
-#'   position with the columns during tabulation. This functions accepts the same
-#'   parameters as [analyze] like `afun` and `format`. For further information see
-#'   [`additional_fun_params`].
+#' Generate rows analyzing different variables across columns
 #'
-#' @export
+#' @inheritParams lyt_args
+#' @param afun (`function` or `list`)\cr function(s) to be used to calculate the values in each column. The list
```

Comment:
is not this in lyt_args?

Reply:
It's a bit of a different description compared to `afun` for `analyze` (though I'm not sure that's intentional) so I left it as is.